transport/http2_server.go

```diff
@@ -143,6 +146,10 @@ func (t *http2Server) operateHeaders(hDec *hpackDecoder, s *Stream, frame header
 			hDec.state = decodeState{}
 		}
 	}()
+	if goaway {
+		//Stop creating streams on this transport
+		return nil
+	}
```

Comment:
This part I'm a little unclear on still.  Can you point me to some documentation to explain this more?